dialogue-core/src/main/java/com/palantir/dialogue/core/Channels.java

```diff
@@ -20,20 +20,26 @@
 
 import com.palantir.conjure.java.api.config.service.UserAgent;
 import com.palantir.dialogue.Channel;
+import com.palantir.tritium.metrics.registry.TaggedMetricRegistry;
 import java.util.Collection;
 import java.util.List;
 
 public final class Channels {
 
     private Channels() {}
 
-    public static Channel create(Collection<? extends Channel> channels, UserAgent userAgent) {
+    public static Channel create(
+            Collection<? extends Channel> channels,
+            UserAgent userAgent,
+            TaggedMetricRegistry metrics) {
         List<LimitedChannel> limitedChannels = channels.stream()
                 .map(ConcurrencyLimitedChannel::create)
                 .collect(toList());
 
         return new UserAgentChannel(
-                new RetryingChannel(new QueuedChannel(new RoundRobinChannel(limitedChannels))),
+                new InstrumentedChannel(
```

Comment:
think you want to stick this around the bottom most channel so we record just time observed in the remote server. then we can other metrics for things like time spent in the queue

Reply:
we'd probably benefit from a method which flattens the channel initialization

Reply:
can you elaborate, @markelliot ?

Reply:
specifically, most of the channels have different arguments, so it's not clear to me how to canonicalize anything here. let me know.